onnxruntime/python/tools/transformers/benchmark_helper.py

```diff
@@ -215,6 +215,7 @@ def output_fusion_statistics(model_fusion_statistics, csv_filename):
 
 def inference_ort(ort_session, ort_inputs, result_template, repeat_times, batch_size):
     result = {}
+    timeit.repeat(lambda: ort_session.run(None, ort_inputs), number=1, repeat=5) # Dry run
```

Comment:
Could we add a parameter (like warm_up_times, default=0) so that we can enable/disable dry run?

Reply:
parameter added and used only when trt is enabled
(actually I found trt needs dry run per test otherwise the first few runs with updated sized inputs are really slow)